Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM maven:3.8.3-openjdk-17
```

Comment:
Doesn't seem like this file is necessary and can probably be removed (according to the description of `lib` in the assignment)

Reply:
I don't think containerizing lib is a bad idea in this case